gui/src/icons/input.tsx

```diff
@@ -2,7 +2,7 @@ import { SvgIcon, SvgIconProps } from "@mui/material";
 
 export const Input = (props: SvgIconProps) => (
     <SvgIcon {...props} viewBox="0 0 16 16">
-        <g stroke="currentColor" stroke-linecap="round" stroke-linejoin="round">
+        <g stroke="currentColor" strokeLinecap="round" strokeLinejoin="round">
```

Comment:
You can remove those properties in the two SVG fragments.
This image is so small that this would make no difference.